jstp/src/main/java/com/metarhia/jstp/connection/RestorationPolicy.java

```diff
@@ -3,15 +3,16 @@
 import java.util.Queue;
 
 /**
- * Created by lundibundi on 2/20/17.
+ * Restoration policy interface for {@link Connection}
  */
 public interface RestorationPolicy {
 
   /**
    * Called after restoration handshake so that this client may resend buffered messages
    *
    * @param connection connection to work with
-   * @param sendQueue connection buffer of messages that was sent but was not received
+   * @param sendQueue  connection buffer of messages that was sent but was not received
```

Comment:
connection buffer of messages that were sent but was not confirmed to be received on the other end of a connection